### DISCUSSION TITLE
Bibliometrics/ Statistics: View / Download Counts for Publications

### DISCUSSION THREAD

**Discussion by IULibScholComm** [quarto-dev/quarto-cli#8995] [Feature Requests]
### Description

Dear Quarto Team,
May I request the following feature, if such a feature is possible: 

the ability to get / record the # of views of a HTML publication (book, article, etc.), and / or the # of downloads of that publication's downloadable formats, such as PDF, EPUB, etc.? Learning how many times a book, article, etc. has been viewed / downloaded would help with research impact.

No worries if this is not possible, but I thought I would ask, as it would be helpful. 
Thank you!

**Comment by mcanouil** [comment]
That's statistics tools such as Google Analytics, see <https://quarto.org/docs/websites/website-tools.html#google-analytics>.

Note that you can use any web statistics tools and include that in your website (you can take inspiration from the following extension to add Google Ad Sense: https://github.com/coatless-quarto/adsense).

Very unlikely Quarto will integrate statistics tools by default.

**Reply by IULibScholComm** [reply]
Cool--thank you!
